README.rst

```diff
@@ -6,13 +6,166 @@ App for tracking sprint commitments, vacations and spillovers.
+6. Remaining time, which is the result of `Goal` - `Committed`.
+
+User's dashboard
+~~~~~~~~~~~~~~~~
+This view shows all assigned (as `Assignee` or `Reviewer 1`) tickets of the user with:
+
+1. Task's key (you can hover over it to see the ticket's name)
+2. User's role
+3. Current status of the ticket
+4. Remaining time for the current user
+5. Sprint indicator (active or future one)
+6. Epic/Story indicator
+
+Caching
+~~~~~~~
+After you refresh the board for the second time, you’ll immediately see cached data **and a spinner showing that it’s being reloaded**. This makes using the dashboard much smoother.
+
+
+Completing the sprints
+~~~~~~~~~~~~~~~~~~~~~~
+In case when users needs to schedule tickets for sprints that haven’t been created yet, they can press the `Create Next Sprint` to create a new one for the currently viewed cell.
+
+Completing the sprints
+~~~~~~~~~~~~~~~~~~~~~~
+To complete the sprint, you need to have `Staff status` permissions.
+The main idea behind this is that they are not shared by cells - you need to have separate sprint for each one. You can press the `Complete Sprint` button on the cell's dashboard to schedule a Celery task with the following pipeline:
+
+1. Upload spillovers.
+    This uploads all spillovers to the `GOOGLE_SPILLOVER_SPREADSHEET`. The following rows are filled in the spreadsheet:
+
+    a) Ticket
+        The key of the ticket.
+    b) Status
+        The status of the ticket at the moment of ending the sprint.
+    c) Sprints
+        The active sprint (the one that is currently being ended).
+    d) Assignee
+        The assignee, for whom the spillover is being counted.
+    e) Reviewer 1
+    f) Reviewer 2
+    g) Reporter
+    h) Story Points
+    i) Estimated time
+        The initial estimation of the ticket (in hours).
+    j) Remaining time
+        The remaining time for the ticket (in hours).
+    k) Reason for the spillover
+        The reason of the spillover is retrieved from the comments made within the active sprint. The assignees should provide it with a comment matching the following regexp: ```[~{JIRA_BOT_USERNAME}\]: <spillover>(.*)<\/spillover>```. In case of multiple occurrences of comments matching this regexp, only the last one is taken into account. In case of no occurrences of such comments, the Jira bot will create a comment defined in `SPILLOVER_REMINDER_MESSAGE`.
+
+    If the users have achieved the clean sprint (without spillovers), they can post some hints on the ticket with the `SPRINT_MEETINGS_TICKET` name by adding a comment matching the spillover reason regexp (provided above). In case of no such comment, they will be reminded on the ticket with `SPILLOVER_CLEAN_HINTS_MESSAGE` comment. It's possible to disable the pings for specific users by adding them to `SPILLOVER_CLEAN_SPRINT_IGNORED_USERS` (this can be useful for people that are members of multiple cells, as they will be pinged on each cell-specific ticket).
```

Comment:
@Agrendalath Typo: "achieved the clean sprint" should be "achieved a clean sprint".

Also, nit: I think it would be nice to replace "users" with "team members" :)

Reply:
Yep, sounds nicer :)